packages/events/src/router/event/index.ts

```diff
@@ -283,15 +284,31 @@ export const eventRouter = router({
         })
     })
   }),
-  list: publicProcedure
+  list: systemProcedure
     .use(requiresAnyOfScopes(ACTION_ALLOWED_SCOPES[ActionType.READ]))
     .output(z.array(EventIndex))
     .query(async ({ ctx }) => {
+      if (ctx.userType === TokenUserType.SYSTEM) {
+        return getIndexedEvents(ctx.system.id)
+      }
       const userId = ctx.user.id
       return getIndexedEvents(userId)
     }),
   search: publicProcedure
     .use(requiresAnyOfScopes(CONFIG_SEARCH_ALLOWED_SCOPES))
     .input(QueryType)
-    .query(async ({ input }) => getIndex(input))
+    .query(async ({ input }) => getIndex(input)),
+  import: systemProcedure
+    .use(requiresAnyOfScopes([SCOPES.RECORD_IMPORT]))
+    .meta({
+      openapi: {
+        summary: 'Import full event record',
+        method: 'POST',
+        path: '/events/import',
+        tags: ['Events']
```

Comment:
```suggestion
        tags: ['events']
```

openapi [docs](https://swagger.io/docs/specification/v3_0/grouping-operations-with-tags/) use lowercase